app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/labtest/LabTestScreen.kt

```diff
@@ -0,0 +1,394 @@
+        Spacer(modifier = Modifier.size(8.dp))
+        Column {
+            Text(
+                text = title,
+                color = blue,
+                style = MyHealthTypography.body2.bold()
+            )
+
+            clickableText?.let {
+                MyHealthClickableText(
+                    style = MyHealthTypography.body2.copy(color = blue),
+                    fullText = body,
+                    clickableText = it,
+                    action = onClickFaq,
+                    clickableStyle = SpanStyle(
+                        color = primaryBlue,
+                        textDecoration = TextDecoration.Underline,
+                        fontWeight = FontWeight.Bold
+                    )
+                )
+            } ?: run {
+                Text(
+                    style = MyHealthTypography.body2.copy(color = blue),
+                    text = body,
+                )
+            }
+        }
+    }
+}
+
+@Composable
+private fun LabTestUi(labTestDetail: LabTestDetail, onClickFaq: () -> Unit) {
+    LabTestHeaderUi(
+        labTestDetail.header,
+        labTestDetail.summary?.let { stringResource(id = it) },
+        onClickFaq
+    )
+
+    LabTestItemUi(
+        labTestDetail.title1,
+        labTestDetail.testName.orPlaceholderIfNullOrBlank(),
+    )
+
+    LabTestResultItemUi(labTestDetail)
+
+    LabTestItemUi(
+        labTestDetail.title3,
+        labTestDetail.testStatus?.let { stringResource(id = it) },
+    )
+
+    LabTestDivider()
+}
+
+@Composable
+private fun LabOrderUi(labTestDetail: LabTestDetail) {
+    LabTestItemUi(
+        labTestDetail.title1,
+        labTestDetail.timelineDateTime.orPlaceholderIfNullOrBlank(),
+    )
+
+    LabTestItemUi(
+        labTestDetail.title2,
+        labTestDetail.orderingProvider.orPlaceholderIfNullOrBlank(),
+    )
+
+    LabTestItemUi(
+        labTestDetail.title3,
+        labTestDetail.reportingSource.orPlaceholderIfNullOrBlank(),
+    )
+
+    LabTestDivider()
+}
+
+@Composable
+private fun LabTestHeaderUi(title: Int?, body: String?, onClickFaq: () -> Unit) {
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(top = 16.dp)
+    ) {
+        title?.let {
+            Text(
+                text = stringResource(id = it),
+                style = MyHealthTypography.body2.bold()
+            )
+        }
+
+        body?.let {
+            MyHealthClickableText(
+                style = MyHealthTypography.body2,
+                fullText = it,
+                clickableText = stringResource(R.string.learn_more),
+                action = onClickFaq,
+                clickableStyle = SpanStyle(
+                    color = primaryBlue,
+                    textDecoration = TextDecoration.Underline,
+                    fontWeight = FontWeight.Bold
+                )
+            )
+        }
+    }
+}
+
+@Composable
+private fun LabTestItemUi(title: Int?, body: String?) {
```

Comment:
I would suggest to have this in its own separate file with its own preview.
It should stay in the same package. if we find a use case of reuse we will take this to the component package

Reply:
done